openshift-kube-apiserver/filters/deprecatedapirequest/apiaccess_count_controller_test.go

```diff
@@ -607,6 +612,16 @@ func TestPersistRequestCountForAllResources(t *testing.T) {
 				)),
 			},
 		},
+		{
+			name: "UserError",
+			existing: []runtime.Object{
+				apiRequestCount("test-v1-group"),
+			},
+			expected: []*apiv1.APIRequestCount{
+				apiRequestCount("test-v1-group"),
+			},
+			logContains: "name must be of the form 'resource.version.group'",
```

Comment:
wouldn't be better to validate the content of `countsToPersist` instead?

Reply:
Updated test. Should ignore invalid resource (i.e. not panic) and still process other valid resources.